B827EBFFFE7367F6.json

```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "B827EBFFFE7367F6", 
+		"servers": [ 
+			{ 
+				"server_address": ""router.eu.thethings.network", 
```

Comment:
There is an extra double quote here that is making it invalid. Please fix and we can merge